about.html

```diff
@@ -0,0 +1,137 @@
+              </li>
+              <li class="nav-item p-2">
+                <a class="nav-a active-nav" href="index.html">Home</a>
+              </li>
+              <li class="nav-item p-2">
+                <a class="nav-a" href="about.html">About</a>
+              </li>
+              <li class="nav-item p-2">
+                <a class="nav-a join-us" href="#">Join Us</a>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </nav>
+
+      <section id="main-header">
+        <h3 class="main-header-h3 d-md-none">"Hello! Sharing world"</h3>
+        <h1 class="main-header-h1">ProCoder Full-Stack Diploma 2021</h1>
+        <p class="main-header-desc">The Global Diploma brings together the community of experts, academics, and activites who comprise the ProCoder affiliate network in a different countery every two. This year, we hope to expand our invitation list including organizations and individuals</p>
+        <p class="main-header-p">Please contact us per Email for any further questions about ProCoder Diploma 2021!</p>
+        <p class="main-header-mail">ProCoder@gmail.com</p>
+      </section>
+    </header>
+
+    <main>
+      
+
+      <section id="date">
+        <h2 class="date-h2">ProCoder Full-Stack Diploma 2021</h2>
+        <p class="date-p">The logo of CC ProCoder was decided through the logo competition from 8. June to 7. July.</p>
+        <img class="date-img" src="images/logo1.jpg" alt="ProCoder logo">
+      </section>
+      <hr>
+      <section id="past">
+        <h2 class="past-h2">See The Past ProCoder Full-Stack Diplomas</h2>
+        <p class="past-p">Take a look at the past two years of our diplomas courses and our students.</p>
+        <div class="past-images row">
+          <div class="img-container col-md-5 offset-md-1 me-md-3">
+            <img class="past-img" src="images/class1.jpg" alt="">
+            <div class="layer">
+              <p class="text-past">ProCoder Diploma 2019</p>
+            </div>
+          </div>
+          <div class="img-container col-md-5">
+            <img class="past-img" src="images/class2.jpg" alt="">
+            <div class="layer">
+              <p class="text-past">ProCoder Diploma 2020</p>
+            </div>
+          </div>
+        </div>
+        
+      </section>
+
+      <section id="partners">
+        <h2 class="partners-h2">Partner</h2>
+        <ul class="partners-companies list-unstyled">
+          <li><img src="images/mozilla.png" alt="mozilla"></li>
+          <li><img src="images/google.png" alt="google"></li>
+          <li><img src="images/naver.png" alt="naver"></li>
+          <li><img src="images/damkakao.png" alt="daumkako"></li>
+          <li><img src="images/airbnd.png" alt="airbnb"></li>
+        </ul>
+    </section>
+    </main>
+
+    <footer>
+      <img class="footer-img" src="images/logo.jpg" width="100" alt="ProCoder logo">
+      <p>2021 ProCoder All Rights Reserved</p>
+    </footer>
```

Comment:
- [ ] According to the design guidelines, in the `desktop` screen version of the `About` page the footer has a dark background.
- [ ] According to the design guidelines, the `partners` section shouldn't be present in the `desktop` screen version of the `About` page.
Reference Design
![image](https://user-images.githubusercontent.com/33767951/142299040-28d1c01f-0203-4197-8a92-019d2bd3d10d.png)

Reply:
Done